inference-engine/include/ie_blob.h

```diff
@@ -754,7 +776,16 @@ class TBlob : public MemoryBlob {
      * @brief Returns handle to the stored data.
      */
     void* getHandle() const noexcept override {
-        return _handle;
+        return _handle.get();
+    }
+
+    TBlob(const TBlob& origBlob, const ROI& roi) :
```

Comment:
Why do we need this constructor? can we use `createROI` instead of this?

Reply:
It is an implementation for `createROI`, it calls this constructor.